test/resolver.spec.js

```diff
@@ -228,7 +228,7 @@ describe('IPLD Format resolver (local)', () => {
   })
 
   it('resolver.isLink for non valid CID', (done) => {
-    resolver.isLink(dataLinksNodeBlock, 'Links/0/Name', (err, link) => {
+    resolver.isLink(dataLinksNodeBlock, 'Links/0/Tsize', (err, link) => {
```

Comment:
Why the change in the test? It should also be false for /Name.

Reply:
True, but maybe it should check both then. Checking only name doesn't cover the general (non-blank-value) case.